go/common/version/version.go

```diff
@@ -54,18 +54,18 @@ var (
 	// the runtime.
 	//
 	// NOTE: This version must be synced with runtime/src/common/version.rs.
-	RuntimeProtocol = Version{Major: 0, Minor: 8, Patch: 0}
```

Comment:
This doesn't need to change?

Reply:
Yeah, I wasn't sure about which structures are in which version notions. So `RuntimeProtocol` isn't changed, but `CommitteeProtocol` and `BackendProtocol` should be bumped when changing runtime descriptor?

Reply:
lol I also bumped the `BackendProtocol` version, but someone also bumped it in master so after rebase it was gone :P It's there now :+1: